style.css

```diff
@@ -406,3 +426,283 @@ textarea[type="text"] {
   margin-bottom: 45px;
   margin-left: 24px;
 }
+
+#works-section-desktop {
+  display: none;
+}
+
+@media (min-width: 500px) {
```

Comment:
- [ ] Please, use media queries for the Desktop breakpoint in the CSS file.(min-width > 992px - 1100px), for example `@media screen and (max-width: 1080px) {` 🐶